react/src/views/Box/BoxView.tsx

```diff
@@ -283,7 +282,23 @@ function BTBox() {
         newState,
       },
       // refetchQueries: [refetchBoxByLabelIdentifierQueryConfig(labelIdentifier)],
-    });
+    })
+      .then((res) => {
+        notificationVar({
+          title: `Box ${labelIdentifier}`,
+          type: res?.errors ? "error" : "success",
+          message: res?.errors
+            ? "Error while trying to update the box state"
```

Comment:
"Error: Could not update the box state to {state}." should be the error. Then what happens? Does the whole action fail? Should the user try again?

Reply:
@aerinsol yes the whole action fails